### DISCUSSION TITLE
Pagetual on wallpaper engine workshop browser ?

### DISCUSSION THREAD

**Discussion by Sko0b** [hoothin/UserScripts#473] [Ideas]
Hi, I'm using Adguard to run pagetual anywhere, android, windows, steam client 😀 . The best script ever

Don't know if I'm in the right channel but I would like you to include Wallpaper Engine Worhshop browser. I'm using the .exe , would it be possible and how. Many thx buddy

**Comment by hoothin** [comment]
Hi, I have no idea, but is that "Wallpaper Engine Worhshop browser.exe" running on a webview?
Pagetual is a script running on web browser ( steam client also has a web browser inside - Electron based on chromium ), any webview won't support script until it make a script platform.

**Reply by Sko0b** [reply]
Yes, im using adguard has a script host platform. Pardon my language im french. Hard to explain. 
Adguard lifetime key on stacksocial.com totally worth it. If u wanna try it. Runs script on any program's.. 

I ddnt knew for webview .... its early il write back around 10 am now its 4 am ...

**Reply by Sko0b** [reply]
Wallpaper engine on steam
